test/lib/config.js

```diff
@@ -31,7 +31,21 @@
         window.__ECHARTS__DEFAULT__RENDERER__ = params.__RENDERER__;
     }
     if (params.__COARSE__POINTER__) {
-        window.__ECHARTS__DEFAULT__COARSE_POINTER = params.__COARSE__POINTER__;
+        switch (params.__COARSE__POINTER__) {
+            case true:
```

Comment:
The values parsed from location search params won't be in other types except for the `string` type. So `case true` & `case false` isn't necessary.